.gitignore

```diff
@@ -4,3 +4,4 @@
 .phpunit.result.cache
 composer.lock
 phpunit.xml
+.idea/
```

Comment:
It looks like people in this library only gitignore language specific items, not environment specific, so let's not contribute this.